tests/components/sensor/test_recorder.py

```diff
@@ -154,11 +159,13 @@ def test_compile_hourly_statistics_unsupported(hass_recorder, caplog, attributes
     [
         ("energy", "kWh", "kWh", 1),
         ("energy", "Wh", "kWh", 1 / 1000),
-        ("monetary", "€", "€", 1),
+        ("monetary", "EUR", "EUR", 1),
```

Comment:
I'm not a reviewer, just interested in the gas support. That said, this monetary change seem unrelated to the PR.

Reply:
It was just a wrong value in the tests as it is supposed to be ISO 4217 now.